examples/github_actions_orchestration/README.md

```diff
@@ -0,0 +1,74 @@
+# 🏃 Run pipelines in GitHub Actions
+
+# 🖥 Run it locally
+
```

Comment:
There should be a motivating introduction sentence that explains why this orchestrator is useful ... 


Also the `# Run it locally` seems misleading. Probably should be renamed or removed completely